### ISSUE TITLE
Review TripleDES warning

### ISSUE THREAD

**Issue by tcompa** [fractal-analytics-platform/fractal-server#1672]
```
/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/site-packages/paramiko/pkey.py:100: CryptographyDeprecationWarning: TripleDES has been moved to cryptography.hazmat.decrepit.ciphers.algorithms.TripleDES and will be removed from this module in 48.0.0.
```

**Comment by ychiucco** [comment]
https://github.com/paramiko/paramiko/issues/2419

**Comment by tcompa** [comment]
This should be fixed as of paramiko 3.4.1 (https://www.paramiko.org/changelog.html). Let's update our poetry.lock, and verify that the warning is not there any more.

**Comment by ychiucco** [comment]
Downgrade `paramiko`
```sh
$ poetry lock --no-update
Resolving dependencies... (0.7s)

Writing lock file

$ poetry install -E postgres
Installing dependencies from lock file

Package operations: 0 installs, 1 update, 0 removals

  - Downgrading paramiko (3.4.1 -> 3.4.0)

Installing the current project: fractal-server (2.3.10)
```

Run some tests
```sh
$ poetry run pytest tests/no_version
...
====================================================================================================================================================== warnings summary ======================================================================================================================================================
../../../../Library/Caches/pypoetry/virtualenvs/fractal-server-zZ2T8p1a-py3.10/lib/python3.10/site-packages/passlib/pwd.py:16
  /Users/yuri/Library/Caches/pypoetry/virtualenvs/fractal-server-zZ2T8p1a-py3.10/lib/python3.10/site-packages/passlib/pwd.py:16: DeprecationWarning: pkg_resources is deprecated as an API. See https://setuptools.pypa.io/en/latest/pkg_resources.html
    import pkg_resources

../../../../Library/Caches/pypoetry/virtualenvs/fractal-server-zZ2T8p1a-py3.10/lib/python3.10/site-packages/paramiko/pkey.py:100
  /Users/yuri/Library/Caches/pypoetry/virtualenvs/fractal-server-zZ2T8p1a-py3.10/lib/python3.10/site-packages/paramiko/pkey.py:100: CryptographyDeprecationWarning: TripleDES has been moved to cryptography.hazmat.decrepit.ciphers.algorithms.TripleDES and will be removed from this module in 48.0.0.
    "cipher": algorithms.TripleDES,

../../../../Library/Caches/pypoetry/virtualenvs/fractal-server-zZ2T8p1a-py3.10/lib/python3.10/site-packages/paramiko/transport.py:259
  /Users/yuri/Library/Caches/pypoetry/virtualenvs/fractal-server-zZ2T8p1a-py3.10/lib/python3.10/site-packages/paramiko/transport.py:259: CryptographyDeprecationWarning: TripleDES has been moved to cryptography.hazmat.decrepit.ciphers.algorithms.TripleDES and will be removed from this module in 48.0.0.
    "class": algorithms.TripleDES,
...
```

Now upgrade `paramiko` and run the same tests
```
$ poetry install -E postgres
Installing dependencies from lock file

Package operations: 0 installs, 1 update, 0 removals

  - Updating paramiko (3.4.0 -> 3.4.1)

Installing the current project: fractal-server (2.3.10)

$ poetry run pytest tests/no_version
```
And there is no `TripleDES` warning.
